slave/process-scim/lib/process-scim.example

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+'''
+EXAMPLE of python script for processing SCIM files from Perun
+'''
+
+USERS_FILE = '/var/spool/perun/scim-service-users.scim'
+GROUPS_FILE = '/var/spool/perun/scim-service-grops.scim'
```

Comment:
These two files should be taken from input of this script (in execution of this script on bash side), instead of duplicate their path in bash and in python.

Reply:
Variables are passed to the external script via command line arguments.